src/nl/release-notes/20.frontend.md

```diff
@@ -0,0 +1,28 @@
+---
+title: MyParcel UI notes
+sidebar: auto
+contributors: false
+editLink: false
+---
+Hieronder kan je de release notes vinden van de MyParcel UI. Elke maand doen we twee releases voor de UI. Relevante dingen zullen hieronder benoemd worden. Als er vragen over een release zijn, neem dan contact op via [MyParcel dev support Slack](https://join.slack.com/t/myparcel-dev/shared_invite/enQtNDkyNTg3NzA1MjM4LWQ5MWE5MTQ3MDg4YjU5NzdjYjk0OTY1ZDJiYjU5YzJjNzk3Yzk3NGY0OWFkZDU4MDYwZDEyZDlhZTgzOWM1MjI/).
```

Comment:
ik denk niet dat we "UI" gebruikers die release notes niet begrijpen naar de dev slack moeten verwijzen...

Reply:
Ik verkeer(de) onder de indruk dat developers de doelgroep[ is van de release notes